plugins/system/redirect/redirect.php

```diff
@@ -285,6 +285,17 @@ private static function doErrorHandling($error)
 				// In case the url contains double // lets remove it
 				$destination = str_replace(JUri::root() . '/', JUri::root(), $dest);
 
+				// Always count redirect hits
+				$redirect->hits++;
```

Comment:
```suggestion
				++$redirect->hits;
```

Im not sure about others, but incrementing on a line by its own, I have always used the pre and not post version of `++` (Yes I understand the difference, and yes, its probably ok this way

Looking at more professional projects and they use `++$value`

Joomla has both versions, and its not documented in the coding standards, except by accident in the `A do-while Example`

Reply:
I see its RTC already - so I'll delete this comment as not to off-rail it.